source/NVDAObjects/UIA/__init__.py

```diff
@@ -515,10 +515,19 @@ def _get_description(self):
 			return ""
 
 	def _get_keyboardShortcut(self):
+		ret = ""
 		try:
-			return self.UIAElement.currentAccessKey
+			ret += self.UIAElement.currentAccessKey
 		except COMError:
-			return None
+			pass
+		if ret != "":

```

Comment:
tiny nit: It's more Pythonic to just do:

`if ret:`

A string tests as true if it is non-empty.